test/integration/UrlQueryPatternTest.php

```diff
@@ -215,11 +227,13 @@ public function testQueryPatternWithCustomLangAliases()
         TestUtils::setWovnIni("{$this->docRoot}/wovn.ini", array(
             'url_pattern_name' => 'query',
             'supported_langs' => array_keys($langs),
-            'custom_lang_aliases' => $langs
+            'custom_lang_aliases' => $langs,
+            'query_canonical_significance' => true
         ));
 
-        $content_without_html_swapper = '<html lang="en">'.
+        $content_without_html_swapper_custom_en = '<html lang="en">'.
         '<head>'.
+        '<link rel="canonical" href="http://localhost/index.html?wovn=custom_en">'.
```

Comment:
When lang alias is available for default lang, it is incoporated into canonical url.